pep-3156.txt

```diff
@@ -147,6 +148,15 @@ wait for a Future to complete by adding a callback to the Future.
 Likewise, the scheduler offers an operation to suspend a coroutine
 until a callback is called.
 
+In case of any other async framework cannot just use Future and Task
+as is for some reason the framework may reimplement
+``loop.create_future()`` and ``loop.create_task()`` calls by returning
+an object which implements a superset of Future/Task interfaces but
+adds a new functionality required by the framework.
+
+``loop.set_task_factory()`` may be useful as well for adopting asyncio
```

Comment:
`set_task_factory` seems dangerous in conjunction with overridable `create_task` - a task factory that works with asyncio loops probably won't work with tornado loops because tornado overrides `create_task`. This seems like both too much flexibility (lots of ways to change the task implementation) and too little (can't mix multiple task implementations per loop).